backend/main/server/app.go

```diff
@@ -568,9 +572,12 @@ func (a *App) createVoteForProposal(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	s.FlowAdapter = a.FlowAdapter
```

Comment:
I'm getting a compile error on this line: `main/server/app.go:575:4: s.FlowAdapter undefined (type Strategy has no field or method FlowAdapter)`

Reply:
That should be fixed in the last push